File: CorejavaExample/Day02Assignment/src/com/sonata/Tshirt.java
package com.sonata;

public class Tshirt 
{
	String color;
	String material;
	String design;
	String size;
	Tshirt(String c,String m,String d,String s)
	{
		this.color= c;
		this.material= m;
		this.design=d;
		this.size=s;
	}
Tshirt(){}

public void Display()
{
	System.out.println(color);
	System.out.println( design);
	System.out.println( material);
	System.out.println( size);
}
	public static void main(String[] args) 
	{
		Tshirt t1=new Tshirt("black","wool","sweat tshirt","small");
		Tshirt t2=new Tshirt("blue","cotton","round neck","large");
		Tshirt t3=new Tshirt("red","denim","collar less","extra-large");
		System.out.println("first Tshirt:");
		t1.Display();
		System.out.println("second Tshirt:");
		t2.Display();
		System.out.println("third Tshirt:");
		t3.Display();
	}



}

File: Typescript_2021/T_shirt.ts
class Tshirt{

    color:string="blue";

    material:string="Rayon";

     design:string="VShaped"

     constructor(i:string,n:string,s:string)

    {

    this.color=i;

    this.material=n;

    this.design=s;

    }

    display() {

    console.log(this.color); console.log(this.material); console.log(this.design);

    }

    }

    var small=new Tshirt("Red","Polyster","Round")

     small.display();

    var large=new Tshirt("Pink","Poly","V Shaped");

    large.display();

    var xs=new Tshirt("Brown","Nylon","Sleeves");

    xs.display();